src/content/docs/en/reference/experimental-flags/serialized-configuration.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Experimental serialized configuration
+sidebar:
+  label: Serialized configuration
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro';
+
+<p>
+
+  **Type:** `boolean`<br />
+  **Default:** `false`<br />
+
+  <Since v="5.2.0" />
+</p>
+
+This feature allows to use a new virtual module called `astro:config`. The virtual module exposes a non-exhaustive, serializable, type-safe version of the Astro configuration.
+
+The virtual module exposes two "submodules".
+
+## `astro:config/client`
+
+It exposes information that are **safe** to expose to the client (browser), hence it can be used in any instance of the application. It exposes information such as `trailingSlash`, `build.format`, `i18n`, and more.
+
+## `astro:config/server`
```

Comment:
NB: Did not make suggestions here, but whatever we do in the section above, we should duplicate here.